int8 UNKNOWN = 0
int8 ON = 1
int8 OFF = 2

int8 hotspot_status
int8 wlan0_status
